java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1249,6 +1249,28 @@ public void zrank(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zremrangebyrank(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+
+        assertEquals(0, client.zremrangebyrank(key1, 2, 1).get());
+        assertEquals(2, client.zremrangebyrank(key1, 0, 1).get());
+        assertEquals(1, client.zremrangebyrank(key1, 0, 10).get());
```

Comment:
Call `zrange(key, 0, -1)` to check what remains in the zset after each `zremrangebyrank`

Reply:
Copied tests from Node your call.

Reply:
Please enrich tests

Reply:
Added. Please take a look.